pronotepy/clients.py

```diff
@@ -442,6 +442,47 @@ def information_and_surveys(self, date_from: datetime.datetime = None, date_to:
 
         return info
 
+    def menus(self, date_from: datetime.date, date_to: datetime.date = None) -> List[dataClasses.Menu]:
+        """
+        Get menus between two given points.
+
+        date_from : datetime
+            The first date
+        date_to : datetime
+            The second date. If unspecified to the end of the year.
+
+        Returns
+        -------
+        List[Menu]
+            Menu between two given points
+        """
+        output = []
+
+        if not date_to:
+            date_to = date_from
+
+        first_week = self.get_week(date_from)
+        last_week = self.get_week(date_to)
+
+        first_day = date_from - datetime.timedelta(days=date_from.weekday())
+
+        # getting menus for all the weeks.
+        for week in range(first_week, last_week + 1):
```

Comment:
Replace with a while loop? Since you're already counting the `first_day` anyways...
```py
while first_day <= date_to:
```

Reply:
yes I hadn't thought about it, it's much simpler like that